spec/player_spec.rb

```diff
@@ -0,0 +1,29 @@
+require './lib/player'
+
+RSpec.describe Player do
+  context 'Create player instances' do
+    it 'raise an ArgumentError if an integer was given' do
+      expect { Player.new('23', 'X') }.to raise_error(ArgumentError, 'you can\'t have an Integer as name')
+    end
+
+    it 'raise an ArgumentError if a short name was given' do
+      expect { Player.new('zac', 'O') }.to raise_error(ArgumentError, 'A name should at least contain 4 letters')
+    end
+
+    it 'make sure that an object is create with no errors' do
+      expect(Player.new('Javier', 'X')).to be
+    end
+  end
+
+  context '#inc_score' do
+    it 'return a valid number of increments' do
+      player = Player.new('Zack', 'O')
```

Comment:
* [ ] Instead of using `before_each` to define a variable for the test cases, students should [Use `let` instead](https://github.com/rubocop-hq/rspec-style-guide#let-blocks). ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/Ruby_testing/before_each.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59566098/))

- Use let to define the player variable on line #20

Reply:
thank you for that, I didn't notice that variable.